xblockutils/templates/studio_edit.html

```diff
@@ -93,7 +93,7 @@
             {% endif %}
           </div>
           {% if field.help %}
-            <span class="tip setting-help"> {{ field.help }} </span>
+            <span class="tip setting-help"> {{ field.help|safe }} </span>
```

Comment:
This change isn't fully backwards compatible.  Some existing help string might rely on special characters being escaped, though that seems unlikely in practice.  I can't think of an easy way to verify this for all XBlocks using this library, so I'm fine with the change.